### ISSUE TITLE
fine_tune_batch_norm=true and batch_size=12 gives 'Loss is inf or nan'

### ISSUE THREAD

**Issue by 1453042287** [tensorflow/models#5774]
### System information
- **What is the top-level directory of the model you are using**:
deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:
pypi
- **TensorFlow version (use command below)**:
1.11
- **Bazel version (if compiling from source)**:
none
- **CUDA/cuDNN version**:
CUDA 9.0 / cuDNN 7.3
- **GPU model and memory**:
2 *Geforce 1080 Ti, 11GB
- **Exact command to reproduce**:
CUDA_VISIBLE_DEVICES=5,4 python deeplab/train.py --logtostderr --training_number_of_steps=50000 --train_split="train" --model_variant="mobilenet_v2" --atrous_rates=12 --atrous_rates=24 --atrous_rates=36 --output_stride=8 --decoder_output_stride=4 --dataset="pascal_voc_seg" --tf_initial_checkpoint=deeplab/logs-for-bdd-1-1/model.ckpt-150000 --train_logdir=deeplab/logs-for-bdd-1-2 --dataset_dir=deeplab/datasets/pascal_voc_seg_for_bdd/tfrecord --train_batch_size=12 --num_clones=2 --fine_tune_batch_norm=False --slow_start_step=200
You can collect some of this information using our environment capture script:

### Describe the problem
hello, when i use the model to train bdd100k, after i train the model with a output_stride=16 for about 30k and then i try to train the model with the output_stride=8 for another 30k just like the deeplabv3's paper said, but when i set the fine_tune_batch_norm=False, the loss is not convergent, so the question is dose the fine_tune_batch_norm=False means i must use a small batch_size like 1?

### Source code / logs
INFO:tensorflow:global step 10: loss = 0.2214 (1.030 sec/step)
INFO:tensorflow:global step 20: loss = 0.5442 (0.555 sec/step)
INFO:tensorflow:global step 30: loss = 0.2877 (0.660 sec/step)
INFO:tensorflow:global step 40: loss = 0.6611 (1.050 sec/step)
INFO:tensorflow:global step 50: loss = 0.9771 (1.060 sec/step)
INFO:tensorflow:global step 60: loss = 0.3172 (0.807 sec/step)
INFO:tensorflow:global step 70: loss = 0.6717 (0.571 sec/step)
INFO:tensorflow:global step 80: loss = 0.3134 (0.755 sec/step)
INFO:tensorflow:global step 90: loss = 0.4875 (0.567 sec/step)
INFO:tensorflow:global step 100: loss = 1.0767 (0.551 sec/step)
INFO:tensorflow:global step 110: loss = 0.3748 (1.099 sec/step)
INFO:tensorflow:global step 120: loss = 0.2953 (1.087 sec/step)
INFO:tensorflow:global step 130: loss = 0.2860 (0.818 sec/step)
INFO:tensorflow:global step 140: loss = 0.3499 (0.741 sec/step)
INFO:tensorflow:global step 150: loss = 0.3700 (0.765 sec/step)
INFO:tensorflow:global step 160: loss = 0.3467 (0.515 sec/step)
INFO:tensorflow:global step 170: loss = 0.3890 (0.541 sec/step)
INFO:tensorflow:global step 180: loss = 0.5042 (0.577 sec/step)
INFO:tensorflow:global step 190: loss = 0.4484 (0.965 sec/step)
INFO:tensorflow:global step 200: loss = 0.5706 (0.569 sec/step)
INFO:tensorflow:global step 210: loss = 131.9931 (0.562 sec/step)
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.InvalidArgumentError'>, Loss is inf or nan. : Tensor had Inf values

**Comment by DahuoJ** [comment]
Hi, your pre-training weight settings may be wrong, 
you can try to get --tf_initial_checkpoint=deeplab/logs-for-bdd-1-1/model.ckpt  to solve this problem.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
